### DISCUSSION TITLE
New window placement

### DISCUSSION THREAD

**Discussion by lukaseppler** [rxhanson/Rectangle#1416] [Ideas]
Hi! Really happy with Rectangle. I love the configurable gap between windows and to the screen edge:
![image](https://github.com/rxhanson/Rectangle/assets/10258/b6245daa-b61b-4ca7-9f4a-1f075ea1840f)
But new windows don't adhere to that rule. Any way to influence where new windows land?

**Comment by rxhanson** [comment]
I've avoided implementing this because it would require polling the size/position of all windows, and additionally you would see the visual delay between a window appearing and it getting moved/resized when the polling finds a new window (unless there's a better way that I'm unaware of). It's not something that I have a desire to add in myself, but would be fine merging a good pull request that adds this as a terminal command enabled feature that's not in the UI.
